packages/oceanfront/src/components/Dialog.vue

```diff
@@ -153,7 +153,7 @@ export default defineComponent({
 
       const resizerElement = dialog.value.querySelector('.dialog-resizer')
       const resizer = resizerElement.getBoundingClientRect()
```

Comment:
I think we need optional chaining here - resizerElement?.getBoundingClientRect()